smartsnippets/static/js/smartsnippets/json_snippet_render.js

```diff
@@ -0,0 +1,101 @@
+//<![CDATA[
+(function($) {
+    function runOnLoad(f) {
+        if (runOnLoad.loaded){
+	    f();
+        } else{
+	    runOnLoad.funcs.push(f);
+        }
+    }
+    runOnLoad.funcs = [];
+    runOnLoad.loaded = false;
+    runOnLoad.run = function() {
+        if (runOnLoad.loaded) return;
+
+        for(var i = 0; i < runOnLoad.funcs.length; i++) {
+	    try {
+	        runOnLoad.funcs[i]();
+	    }catch(e) { }
+        }
+        runOnLoad.loaded = true;
+        delete runOnLoad.funcs;
+        delete runOnLoad.run;
+    };
+    if (window.addEventListener){
+        window.addEventListener("load", runOnLoad.run, false);
+    }else if (window.attachEvent){
+        window.attachEvent("onload", runOnLoad.run);
+    }else{
+        window.onload = runOnLoad.run;
+    }
+    runOnLoad(function(){
```

Comment:
Thanks.
I left the `runOnLoad` just in case it is used by some snippet, but this should not be the case.